src/stc/stc.cpp

```diff
@@ -5062,7 +5062,19 @@ void wxStyledTextCtrl::AutoCompUseListCtrl(bool useListCtrl,
                                     currentTextColour);
 }
 
-
+void wxStyledTextCtrl::AutoCompShow(int lenEntered, const wxVector<wxCompletionItem>& list)
+{
+    m_autocompList.clear();
+    m_autocompList = list;
+    //Create correct format for Scintilla
+    wxString itemListString;
+    for(int i=0; i<m_autocompList.size(); i++)
+    {
+        wxCompletionItem cItem = m_autocompList[i];
```

Comment:
```suggestion
        const wxCompletionItem& cItem = m_autocompList[i];
```
no need to copy it needlessly.

Reply:
sure! Thanks